apps/server/src/modules/learnroom/common-cartridge/common-cartridge-file-builder.spec.ts

```diff
@@ -134,4 +142,97 @@ describe('CommonCartridgeFileBuilder', () => {
 			expect(() => element.transform()).not.toThrow();
 		});
 	});
+
+	describe('common cartridge version 3', () => {
+		beforeAll(async () => {
+			const fileBuilder = new CommonCartridgeFileBuilder(fileBuilderOptionsForCCVerions3).addResourceToFile(
+				webContentResourceProps
+			);
+			fileBuilder
+				.addOrganization(organizationProps)
+				.addResourceToOrganization(ltiResourceProps)
+				.addResourceToOrganization(webLinkResourceProps);
+
+			archive = new AdmZip(await fileBuilder.build());
+		});
+
+		it('should use common cartridge version 1.3.0', () => {
+			const manifest = getFileContentAsString('imsmanifest.xml');
+			expect(manifest).toContain(CommonCartridgeVersion.V_1_3_0);
+		});
+	});
+});
+
+describe('CommonCartridgeFileBuilder for export of version 1.3.0', () => {
+	let archive: AdmZip;
+
+	const getFileContentAsString = (path: string): string | undefined => archive.getEntry(path)?.getData().toString();
+	const fileBuilderOptionsForCCVersion3: ICommonCartridgeFileBuilderOptions = {
+		identifier: 'file-identifier for version 3',
+		copyrightOwners: 'Placeholder Copyright',
+		creationYear: 'Placeholder Creation Year',
+		title: 'file-title',
+		version: CommonCartridgeVersion.V_1_3_0,
+	};
+	const organizationPropsForCCVersion3: ICommonCartridgeOrganizationProps = {
+		version: CommonCartridgeVersion.V_1_3_0,
+		identifier: 'organization-identifier',
+		title: 'organization-title',
+		resources: [],
+	};
+	const ltiResourcePropsForCCVersion3: ICommonCartridgeResourceProps = {
+		version: CommonCartridgeVersion.V_1_3_0,
+		type: CommonCartridgeResourceType.LTI,
+		identifier: 'lti-identifier',
+		href: 'lti-identifier/lti.xml',
+		title: 'lti-title',
+		description: 'lti-description',
+		url: 'https://to-a-lti-tool.tld',
+	};
+	const webContentResourcePropsForCCVersion3: ICommonCartridgeResourceProps = {
+		version: CommonCartridgeVersion.V_1_3_0,
+		type: CommonCartridgeResourceType.WEB_CONTENT,
+		identifier: 'web-content-identifier',
+		href: 'web-content-identifier/web-content.html',
+		title: 'web-content-title',
+		html: '<h1>Text Resource Title</h1><p>Text Resource Description</p>',
+	};
+	const webLinkResourcePropsForCCVersion3: ICommonCartridgeResourceProps = {
+		version: CommonCartridgeVersion.V_1_3_0,
+		type: CommonCartridgeResourceType.WEB_LINK_V1,
+		identifier: 'web-content-identifier',
+		href: 'web-content-identifier/web-content.xml',
+		title: 'web-link-title',
+		url: 'https://to-a-web-link.tld',
+	};
+
+	beforeAll(async () => {
+		const fileBuilder = new CommonCartridgeFileBuilder(fileBuilderOptionsForCCVersion3).addResourceToFile(
+			webContentResourcePropsForCCVersion3
+		);
+		fileBuilder
+			.addOrganization(organizationPropsForCCVersion3)
```

Comment:
I don't see a reason to add an organization and resources. `build` works without them, doesn't it?
Additionally I would regroup the describes. One per public method. Within the build describe the two describes for version 1 & 3.